codegen-client/src/main/kotlin/software/amazon/smithy/rust/codegen/client/smithy/ClientRustSettings.kt

```diff
@@ -125,6 +125,8 @@ data class ClientCodegenConfig(
     val eventStreamAllowList: Set<String> = defaultEventStreamAllowList,
     // TODO(SmithyRuntime): Remove this once we commit to switch to aws-smithy-runtime and aws-smithy-runtime-api
     val enableNewSmithyRuntime: SmithyRuntimeMode = defaultEnableNewSmithyRuntime,
+    /** If true, adds `endpoint_url`/`set_endpoint_url` methods to the service config */
```

Comment:
Just curious, is there a case where we want to disable those methods?

Reply:
Yeah, the SDK replaces those methods with alternate implementations that are slightly more sophisticated.